OrbitGl/SamplingReport.cpp

```diff
@@ -28,41 +28,47 @@ void SamplingReport::ClearReport() {
 }
 
 void SamplingReport::FillReport() {
-  const auto& sample_data = profiler_->GetThreadSampleData();
+  const auto& sample_data = profiler_.GetThreadSampleData();
 
-  for (ThreadSampleData* threadSampleData : sample_data) {
-    ThreadID tid = threadSampleData->thread_id;
+  for (ThreadSampleData* thread_sample_data : sample_data) {
+    ThreadID tid = thread_sample_data->thread_id;
 
-    if (tid == SamplingProfiler::kAllThreadsFakeTid && !profiler_->GetGenerateSummary()) continue;
+    if (tid == SamplingProfiler::kAllThreadsFakeTid && !profiler_.GetGenerateSummary()) continue;
 
     SamplingReportDataView thread_report;
-    thread_report.SetSampledFunctions(threadSampleData->sampled_function);
+    thread_report.SetSampledFunctions(thread_sample_data->sampled_function);
     thread_report.SetThreadID(tid);
     thread_report.SetSamplingReport(this);
     thread_reports_.push_back(std::move(thread_report));
   }
+
+  // Refresh the displayed callstacks as they might not be up to date anymore,
+  // for example the number of occurrences or of total callstacks might have
+  // changed (OrbitSamplingReport::RefreshCallstackView will do the actual
+  // update once OrbitApp::FireRefreshCallbacks is called).
+  UpdateDisplayedCallstack();
```

Comment:
I don't think that we need it here: FillReport called only in constructor and selected_address_ and selected_thread_id_ are default values. It means that `UpdateDisplayedCallstack` calls `ClearReport` in this case (and it doesn't change anything as we've just created SamplingReport).

Reply:
Thanks, for the hint. I guess I was about to remove the UpdateReport method, but noticed, that it is not that easy, as I was expecting it.
Anyways, removed that call.